internal/common/eventmanager.go

```diff
@@ -796,6 +796,7 @@ func (p *EventParams) getStringReplacements(addObjectData, jsonEscaped bool) []s
 		"{{VirtualTargetPath}}", p.getStringReplacement(p.VirtualTargetPath, jsonEscaped),
 		"{{FsTargetPath}}", p.getStringReplacement(p.FsTargetPath, jsonEscaped),
 		"{{ObjectName}}", p.getStringReplacement(p.ObjectName, jsonEscaped),
+		"{{ObjectBaseName}}", p.getStringReplacement(strings.TrimSuffix(path.Base(p.ObjectName), path.Ext(p.ObjectName)), jsonEscaped),
```

Comment:
`p.ObjectName` is already the path.Base of the virtual path. path.Ext is already computed as p.Extension